registry.json

```diff
@@ -3372,5 +3372,19 @@
     "telegram": "DeCentN2Madness",
     "operatorName": "Jeff Hales",
     "github": "qbass"
+  },
+  {
+    "ticker": "STEVE",
+    "poolId": "ad46d051c0eb09b8fad1f371d44d1e0d7f928d301e90bd4a4ed33a5a",
+    "social": {
+      "website": "https://gecspools.io",
+      "twitter": "GecsPools",
```

Comment:
Hey, how about we get some activity on Twitter ?

Just to make sure you are a real person.

Reply:
Sure, no problem!

I just posted an update notifying the world that we recently minted our first block during Epoch 284, and that we are hopeful to join the Single Pool Alliance soon. Let me know if that seems okay!